### DISCUSSION TITLE
Chinese tortoise-tts wechat discuss group

### DISCUSSION THREAD

**Discussion by xiaoyangnihao** [neonbjb/tortoise-tts#636] [General]
中文[tortoise-tts]微信交流群，欢迎各位大佬入群交流~  群二维码过期可加个人微信（yanghongbiao686），备注：tortoise-tts

![tortoise](https://github.com/neonbjb/tortoise-tts/assets/14233490/e72cdfe2-1bcb-43ae-8927-7508cc49c2ad)

**Comment by yangbiao11** [comment]
过期了 再发一呢 或者加我微信拉下我也yangbiaogaoshou
